ETSI/Python/_py__VNF_LCM__VNFD_based-on_SOL001_/Process_Terminate_VNF/Tasks/Task_Get_VNF_LCM_Operation_State.py

```diff
@@ -0,0 +1,20 @@
+from msa_sdk.variables import Variables
+from msa_sdk.msa_api import MSA_API
+
+from custom.ETSI.VnfLcmOpOccsSol003 import VnfLcmOpOccsSol003
+
+
+if __name__ == "__main__":
+
+    dev_var = Variables()
+    context = Variables.task_call(dev_var)
+
+    vnfLcmOpOccs = VnfLcmOpOccsSol003(context["mano_ip"], context["mano_port"])
+    vnfLcmOpOccs.set_parameters(context['mano_user'], context['mano_pass'])
+
+    r = vnfLcmOpOccs.vnf_lcm_op_occs_completion_wait(context["vnf_lcm_op_occ_id"])
+    
+    context["operation_state"] = r.json()['operationState']
+
+    ret = MSA_API.process_content('ENDED', f'{context["operation_state"]}', context, True)

```

Comment:
**vnfLcmOpOccs.state** instead of **ENDED** ?

Reply:
both works, but agree let's use vnfLcmOpOccs.state